content/rs/references/rest-api/_index.md

```diff
@@ -20,75 +20,105 @@ For more info, see:
 
-Specify the version in the request URI, as shown in the following table:
+Specify the version in the request [URI](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier), as shown in the following table:
 
 | Request path | Description |
 |--------------|-------------|
 | POST `/v1/bdbs` | A version 1 request for the `/bdbs` endpoint. |
 | POST `/v2/bdbs` | A version 2 request for the `/bdbs` endpoint. |
 
-When an endpoint supports multiple versions, each version is documented on the corresponding endpoint.  For example, the [bdbs request]({{<relref "/rs/references/rest-api/requests/bdbs/" >}}) page documents POST requests for [version 1]({{<relref "/rs/references/rest-api/requests/bdbs/#post-bdbs-v1" >}}) and [version 2]({{<relref "/rs/references/rest-api/requests/bdbs/#post-bdbs-v2" >}}). 
+When an endpoint supports multiple versions, each version is documented on the corresponding endpoint.  For example, the [bdbs request]({{<relref "/rs/references/rest-api/requests/bdbs/" >}}) page documents POST requests for [version 1]({{<relref "/rs/references/rest-api/requests/bdbs/#post-bdbs-v1" >}}) and [version 2]({{<relref "/rs/references/rest-api/requests/bdbs/#post-bdbs-v2" >}}).
 
-## Authentication
+## Headers
 
-Authentication to the API occurs via Basic Auth. Provide your username and password as the basic auth credentials.
+### Requests
 
-All calls must be made over SSL, to port 9443.
+Redis Enterprise REST API requests support the following HTTP headers:
 
-Example request:
+| Header | Supported/Required Values |
+|--------|---------------------------|
+| Accept | `application/json` |
+| Content-Length | Length (in bytes) of request message |
+| Content-Type | `application/json` (required for PUT or POST requests) |
 
-``` bash
-curl -u "demo@redislabs.com:password" \
-    https://localhost:9443/v1/bdbs
-```
+If the client specifies an invalid header, the request will fail with a [`400 Bad Request`](https://www.rfc-editor.org/rfc/rfc9110.html#name-400-bad-request) status code.
 
-By default, the admin user is authorized for access.  Use [role-based access controls]({{< relref "/rs/security/admin-console-security/user-security.md" >}}) and [role permissions]({{<relref "/rs/references/rest-api/permissions/_index.md" >}}) to manage access.
+### Responses
 
-## Common responses
+Redis Enterprise REST API responses support the following HTTP headers:
 
-The following are common responses which may return regardless of the request type.
+| Header | Supported/Required Values |
+|--------|---------------------------|
+| Content-Type | `application/json` |
+| Content-Length | Length (in bytes) of response message |
+
+## Response types and error codes
+
+[HTTP Status Codes](https://www.rfc-editor.org/rfc/rfc9110.html#name-status-codes) describe the result of an API request. This can be `200 OK` if the server accepted the request, or it can be one of many error codes.
```

Comment:
```suggestion
[HTTP status codes](https://www.rfc-editor.org/rfc/rfc9110.html#name-status-codes) indicate the result of an API request. This can be `200 OK` if the server accepted the request, or it can be one of many error codes.
```